### ISSUE TITLE
t/31-trap-fatals.t fails with non-English locale

### ISSUE THREAD

**Issue by eserte** [genehack/moosex-amine#3]
If LC_ALL=de_DE.UTF-8 is set, then the test suite may fail (seen on FreeBSD and Linux systems):

```
#   Failed test 'Unfound path in constructor error message'
#   at t/31-trap-fatals.t line 21.
#                   'Can't open 'foo/bar/baz.pm' for reading: 'Datei oder Verzeichnis nicht gefunden' at /usr/home/eserte/.cpan/build/MooseX-amine-0.5-6DGAyZ/blib/lib/MooseX/amine.pm line 115
# '
#     doesn't match '(?^:No such file or directory)'
# Looks like you failed 1 test of 8.
t/31-trap-fatals.t .............................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/8 subtests 
```

**Comment by genehack** [comment]
Would forcing `$ENV{LC_ALL}` to `en_US.UTF-8` fix it?

**Comment by eserte** [comment]
No, because setting it within the script is to late. Perl already initialized everything using the outer locale.

**Comment by genehack** [comment]
Fixed, slightly differently, in #4. Thanks!
